### PR TITLE
COM-420 add Representatives API functionality

### PR DESCRIPTION
**Pull request by eric-moov**

### CODE REVIEW

**Review by gyrolc — APPROVED** [review]

File: docs/output/representatives.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+
+
+
+
+
+## Types
+### Representative
+
+Describes an individuals who represents the business.
```
Comment (gerep) [inline docs/output/representatives.md:138]: Maybe `individual`?

Comment (eric-moov) [reply docs/output/representatives.md:138]: Yep.  Fixing the typo.

File: docs/output/representatives.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+}
+```
+    {{</ tab>}}{{</ tabs>}}
+
+
+
+
+### RepresentativeName
+
+Represenatative name
```
Comment (gerep) [inline docs/output/representatives.md:204]: Typo

File: docs/output/representatives.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+
+**Properties**
+
+| Property | Type | Description |
+| ---- | ---- | ----------- |
+  | firstName | `string`| Name this person was given. This is usually the the same as first name.  string <= 64 characters |
```
Comment (gerep) [inline docs/output/representatives.md:210]: `the the` for this and the following lines.

File: docs/output/representatives.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+  | middleName | `string`| Name this person was given. This is usually the the same as middle name.  string <= 64 characters |
+  | lastName | `string`| Family name of this person. This is usually the the same as last name.  string <= 64 characters |
+  | suffix | `string`| Suffix of a given name.  string <= 20 characters |
+
+
+
+### RepresentativePhone
+
+Represenatative phone
```
Comment (gerep) [inline docs/output/representatives.md:219]: Typo.

File: docs/output/representatives.md (hunk 1 above)
Comment (eric-moov) [reply docs/output/representatives.md]: Fixed

File: docs/output/representatives.md (hunk 2 above)
Comment (eric-moov) [reply docs/output/representatives.md]: Fixed

**Review by gerep — APPROVED** [review]
I am not a JS person myself but it looks good :+1:

**Pull request merged**